apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.sql

```diff
@@ -12,6 +12,45 @@ WITH
       AND i.open
     GROUP BY
       i.instance_question_id
+  ),
+  latest_submissions AS (
+    SELECT DISTINCT
+      ON (iq.id) iq.id AS instance_question_id,
+      s.id AS submission_id,
+      s.manual_rubric_grading_id
+    FROM
+      instance_questions AS iq
+      JOIN variants AS v ON iq.id = v.instance_question_id
+      JOIN submissions AS s ON v.id = s.variant_id
+    WHERE
+      iq.assessment_question_id = $assessment_question_id
+    ORDER BY
+      iq.id,
+      s.date DESC
+  ),
+  rubric_grading_to_items AS (
+    SELECT
+      rgi.rubric_grading_id,
+      ri.id AS rubric_item_id
+    FROM
+      rubric_grading_items AS rgi
+      JOIN rubric_items AS ri ON rgi.rubric_item_id = ri.id
+  ),
+  rubric_items_agg AS (
+    SELECT
+      ls.instance_question_id,
+      COALESCE(
+        json_agg(to_jsonb(rgti.rubric_item_id)) FILTER (
```

Comment:
Nit, you don't need `to_jsonb` around scalar values:

```suggestion
        json_agg(rgti.rubric_item_id) FILTER (
```

Reply:
Resolved in efd5036.